packages/terra-date-picker/lib/DatePicker.scss

```diff
@@ -0,0 +1,54 @@
+@import '~terra-mixins';
+@import '../node_modules/react-datepicker/dist/react-datepicker.min';
+
+.terra-DatePicker {
+  display: inline-block;
+  min-width: 14.643em; 
+}
+
+.terra-DatePicker-input {
+  border: 0.071em solid $terra-grey-40;
+  box-shadow: inset 0 0.143em 0.143em $terra-grey-20;
+  font-size: 1em;
+  height: 1.143em;
+  line-height: 1.143em;
```

Comment:
Good to know. https://github.com/cerner/terra-core/commit/d1272119a4c8555ed3db41f88ef190d8e0002666